config/prow/cluster/sinker_rbac.yaml

```diff
@@ -49,6 +49,8 @@ rules:
     verbs:
       - delete
       - list
+      - watch
```

Comment:
This is a breaking change, right? Please add to ANNOUNCEMENTS.md that sinker now needs `watch` and `get` access on pods.